Content.Server/Tabletop/Components/TabletopGameComponent.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Content.Shared.ActionBlocker;
+using Content.Shared.Verbs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Tabletop.Components
+{
+    /**
+     * <summary>A component that makes an object playable as a tabletop game.</summary>
+     */
```

Comment:
I don't think I've ever seen any other comments in the codebase styled like this so should probably update it tbh.